core/src/tangram.cpp

```diff
@@ -69,12 +69,8 @@ void initialize(const char* _scenePath) {
 
         loader.loadScene(sceneString, *m_scene, *m_tileManager, *m_view);
 
-        Primitives::setColor(0xffffff);
-        RenderState::configure();
     }
 
-    while (Error::hadGlError("Tangram::initialize()")) {}
```

Comment:
@karimnaaji is going to put a PR for GL_CHECK macro he has in place in fontstash repo.

Because something like is helpful to find gl errors (as was the case on the S3 issue).